zowe-metrics-api/package.json

```diff
@@ -22,6 +22,7 @@
         "express": "~4.16.1",
         "google-spreadsheet": "^3.1.15",
         "googleapis": "^67.0.0",
+        "https": "^1.0.0",
```

Comment:
`https` is a build-in of node.js https://nodejs.org/api/https.html. Think we don't need this definition?